packages/checkout-ui-extensions-react/README.md

```diff
@@ -0,0 +1,94 @@
+# `@shopify/checkout-ui-extensions-react`
+
+This library provides utilities for writing UI extensions for Checkout using [React](https://reactjs.org).
+
+## Installation
+
+```bash
+$ yarn add @shopify/checkout-ui-extensions-react
+```
+
+## Usage
+
+[UI extensions rendering capabilities](../checkout-ui-extensions/documentation/rendering.md) are built on top of a library called [remote-ui](https://github.com/Shopify/remote-ui). One of the powerful aspects of remote-ui is that it can be the target for JavaScript libraries that support custom (non-DOM) rendering. [React](https://reactjs.org) is one such library, and remote-ui provides a [custom renderer for React](https://github.com/Shopify/remote-ui/tree/main/packages/react) that allows you to use all the power of React, while outputting UI mutations that UI extensions can understand.
```

Comment:
```suggestion
[UI extensions](../checkout-ui-extensions/documentation/rendering.md) are built on top of a library called [remote-ui](https://github.com/Shopify/remote-ui). One of the powerful aspects of remote-ui is that it can be the target for JavaScript libraries that support custom (non-DOM) rendering. [React](https://reactjs.org) is one such library, and remote-ui provides a [custom renderer for React](https://github.com/Shopify/remote-ui/tree/main/packages/react) that allows you to use all the power of React, while outputting UI mutations that UI extensions can understand.
```